### PR TITLE
Change reviewers

### PR DESCRIPTION
**Pull request by henriksunity**
It may be easier to review per-commit as there's a bunch of stuff in here.

### CODE REVIEW

File: src/client/containers/PullRequestSummary/ReviewSection/ReviewerList/index.js [hunk 1]
```diff
@@ -29,25 +25,104 @@ const getStateClass = (state: string) => {
...
-        <OverlayTrigger placement="left" overlay={tooltipOverlay}>
-          <i
-            style={{ marginRight: '12px', width: '12px', marginBottom: '6px' }}
-            className={getClass(props.icon, props.approval)}
-          />
-        </OverlayTrigger>
-        {props.reviewers.map(r => r.user.fullName || r.user.username).join(', ')}
-      </span>
-    </li>
+export const getReviews = createSelector(
```
Comment (mariangemarcano) [inline src/client/containers/PullRequestSummary/ReviewSection/ReviewerList/index.js]: and also it seems that we should put the creation of the selectors in ducks/   as we just talk I think it make sense have some clarifying guidelines

File: src/client/containers/PullRequestSummary/ReviewSection/MissingReviewerList/index.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
+/* @flow */
+
+import React from 'react'
```
Comment (henriksunity) [inline src/client/containers/PullRequestSummary/ReviewSection/MissingReviewerList/index.js]: Good point, yes

File: src/client/containers/PullRequestSummary/ReviewSection/ReviewerList/index.js (hunk 1 above)
Comment (henriksunity) [inline src/client/containers/PullRequestSummary/ReviewSection/ReviewerList/index.js]: General selectors should definitely go in ducks/; control-specific ones could either go there or locally. I don't feel too strongly either way. Let's discuss when Kate's back.

File: src/client/containers/PullRequestSummary/ReviewSection/MissingReviewerList/index.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+export type MissingReviewerListType = {
+  missingReviewers: PullRequestMissingReviewerType,
+  addReviewer: Function,
+  id: string,
+}
+
+// TODO: copied from ReviewSection - should probably be abstracted out somewhere
+const subHeader = text => (
+  <div style={{ color: '#8c8c8c', fontSize: '13px', marginTop: '1em' }}>
```
Comment (henriksunity) [inline src/client/containers/PullRequestSummary/ReviewSection/MissingReviewerList/index.js]: Yes, but until we do, it's probably easiest to just keep it inline.

File: src/client/ducks/pullrequests/queries/pullRequestAddReviewers.graphql [hunk 4]
```diff
@@ -0,0 +1,5 @@
+mutation AddReviewers($pullRequestId: Int!, $reviewers: [UserInput]!) {
+  addReviewers(pullRequestId: $pullRequestId, reviewers: $reviewers) {
+    ok
+  }
+}
```
Comment (henriksunity) [inline src/client/ducks/pullrequests/queries/pullRequestAddReviewers.graphql]: Thought I'd removed both of those files. I'll remove them in an extra commit.

File: src/client/containers/PullRequestSummary/ReviewSection/ReviewerSelection/index.js [hunk 5]
```diff
@@ -3,178 +3,131 @@
...
 
 class ReviewerSelection extends Component {
   /* eslint-disable react/sort-comp */
 
   constructor(props: ReviewersProps) {
     super(props)
     this.state = {
-      usersAfterSearch: this.props.users,
-      query: '',
+      value: [],
```
Comment (henriksunity) [inline src/client/containers/PullRequestSummary/ReviewSection/ReviewerSelection/index.js]: Slightly confusing, I agree. Was to keep consistent with the naming of the select control.

File: src/client/ducks/fetch/index.js [hunk 6]
```diff
@@ -101,3 +101,17 @@ export const fetchActionCreator =
       variables,
       callback,
     })
+
+export const singleFetchActionCreator =
```
Comment (kkateq) [inline src/client/ducks/fetch/index.js]: Slightly odd to change just one parameter type, we should think about better approach in the future ...

**Pull request merged**